openedx_events/learning/signals.py

```diff
@@ -275,3 +276,39 @@
         "course_staff_data": ManageStudentsPermissionData,
     }
 )
+
+# .. event_type: org.openedx.learning.thread.created.v1
+# .. event_name: FORUM_THREAD_CREATED
+# .. event_description: Emitted when a new thread is created in a discussion
+# .. event_data: DiscussionThreadData
+#       Warning: This event is currently incompatible with the event bus, list/dict cannot be serialized yet
+FORUM_THREAD_CREATED = OpenEdxPublicSignal(
```

Comment:
The name is: `FORUM_THREAD_CREATED` so the event type should be: 
`# .. event_type: org.openedx.learning.forum.thread.created.v1` so it matches

Reply:
Updated